writing-docs/author/index.rst

```diff
@@ -103,6 +103,15 @@ As you work, you can freely create new pages and directories and they will autom
 
 Content mapping is determined by :ref:`content mapping configuration files <control-map>` within the control repository. Open an issue on the control repository to discuss the addition of new content, or modify the content mapping files yourself in a pull request if you're also a site coordinator.
 
+.. _pull-request-builds:
+
+Previewing Changes
+------------------
+
+As you work, you'll want to see how the content you're writing will look rendered in context with the entire site. If your content repository is using a :term:`Strider` build, each time you *open a new pull request* or *push new commits to an existing pull request*, Strider will build a preview of your work to a staging environment. Once the build is complete, a bot account will post a comment on your pull request including a link to your personal preview.
+
```

Comment:
> Will you have to provide access to your fork to enable others to view the staged content, or is it publicly accessible to all?

It is publicly accessible to all. There is no authentication in the presenter or the retrieval endpoints of the content service. We do have a [`robots.txt` file in place](https://staging.deconst.horse/robots.txt) that will prevent Google from crawling staging content, but consider staged content as public from a security standpoint.

I _can_ add some form of authentication to protect staged content if it's necessary, but:

1. Staged content is already mapped in the control repository to a public URL, which means that `master` is already live. Even if your content repository is private, everyone on the Internet can already see your final content on the live site anyway.
2. Adding authentication (conditionally on content that's staged from a private content repository) would take a fair amount of infrastructure work that I didn't want to hold this up even more putting in place.

> One thing that I'm not clear about from the technical perspective -- In an environment not powered by an API/content store model, the content is built and then deployed to the server. In the deconst world, the content goes to a content store, and is served via the API. How does the built content get into the content store. Is this part of the Strider build, or a separate process? Maybe I should read the documentation...

The Strider plugin [recognizes when it's building a pull request](https://github.com/deconst/strider-deconst-content/blob/master/worker.js#L8) and [submits content to a staging content endpoint](https://github.com/deconst/strider-deconst-content/blob/master/lib/entry.js#L221-L238) with a temporary API key. It's a regular Strider build other than that - you can see it in the build logs and (now that I've fixed the GitHub status plugin problem) you'll have a link to the build itself on your pull requests, too.

The staging content endpoint is a regular content service endpoint configured to act in "staging mode," which means that it proxies requests for content IDs that aren't present to the real content service after shaving off a "revision ID" (the `build-12345` bit). The staging environment is indeed [already in the docs](https://deconst.horse/developing/staging/) in the developer section.